lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -133,15 +133,16 @@ def self.commmit_changes_and_push_current_branch(commit_message)
         Actions::PushToGitRemoteAction.run(remote: 'origin')
       end
 
-      def self.create_pr_to_main(title, body, repo_name, github_pr_token)
+      def self.create_pr_to_main(title, body, repo_name, github_pr_token, labels)
         Actions::CreatePullRequestAction.run(
           api_token: github_pr_token,
           title: title,
           base: 'main',
           body: body,
           repo: "RevenueCat/#{repo_name}",
           head: Actions.git_branch,
-          api_url: 'https://api.github.com'
+          api_url: 'https://api.github.com',
+          labels: labels
```

Comment:
Yeah, this fastlane action parameter expects an array of strings https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/create_pull_request.rb#L195-L199. So we should fix the comments Cesar posted.

Reply:
Yup I misread it: https://github.com/marketplace/actions/create-pull-request